<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogActivityLog</class>
 <widget class="QDialog" name="DialogActivityLog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>454</width>
    <height>319</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Activity Log</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resources.qrc">
    <normaloff>:/dolphin16</normaloff>:/dolphin16</iconset>
  </property>
  <property name="sizeGripEnabled">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QPlainTextEdit" name="editLog">
     <property name="readOnly">
      <bool>true</bool>
     </property>
     <property name="cursorWidth">
      <number>0</number>
     </property>
     <property name="maximumBlockCount">
      <number>10000</number>
     </property>
     <property name="centerOnScroll">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="labelEntries">
       <property name="text">
        <string>0 log entries</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="saveAs">
       <property name="text">
        <string>Save As...</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
